### PR TITLE
Updated Catalog overview; added Product Sort Order info

### PR DESCRIPTION
**Pull request by ektravel**
# [DEVDOCS-2819](https://jira.bigcommerce.com/browse/DEVDOCS-2819)

## What changed?
* Updated Catalog Overview article
* Added a section on Product Sort Order (Beta)

### CODE REVIEW

File: docs/api-docs/catalog/products-overview.md [hunk 1]
```diff
@@ -1030,6 +1026,32 @@ X-Auth-Token: {{ACCESS_TOKEN}}
 }
 ```
 
+## Product Sort Order (Beta)
+
+[Product Sort Order](add-link-here) allows you to manage the sort order of products displayed on any given category page. Products assigned to multiple storefront categories can have different sort order values per category.
+
+### Product sorting on a storefront 
+
+The Catalog API supports two manually managed methods of product sorting: on a category level and on a product level. If a user combines both sorting methods on a storefront, products are sorted based on the following sorting method priority:
```
Comment (traci707) [inline docs/api-docs/catalog/products-overview.md:1036]: The Catalog API supports two manually managed methods of product sorting: on a category level and a product level. If a user combines both sorting methods on a storefront, the API manually sorts the category products first and then the product-level products.

File: docs/api-docs/catalog/products-overview.md [hunk 2]
```diff
@@ -1030,6 +1026,32 @@ X-Auth-Token: {{ACCESS_TOKEN}}
...
+## Product Sort Order (Beta)
+
+[Product Sort Order](add-link-here) allows you to manage the sort order of products displayed on any given category page. Products assigned to multiple storefront categories can have different sort order values per category.
+
+### Product sorting on a storefront 
+
+The Catalog API supports two manually managed methods of product sorting: on a category level and on a product level. If a user combines both sorting methods on a storefront, products are sorted based on the following sorting method priority:
+
+1. Manually specified sort order on a category level.
+2. Manually specified sort order on a product level. `0` by default.
```
Comment (traci707) [inline docs/api-docs/catalog/products-overview.md:1039]: Removed bullets as I wanted to condense the information. But what is `0` by default?

File: docs/api-docs/catalog/products-overview.md [hunk 3]
```diff
@@ -1030,6 +1026,32 @@ X-Auth-Token: {{ACCESS_TOKEN}}
...
+[Product Sort Order](add-link-here) allows you to manage the sort order of products displayed on any given category page. Products assigned to multiple storefront categories can have different sort order values per category.
+
+### Product sorting on a storefront 
+
+The Catalog API supports two manually managed methods of product sorting: on a category level and on a product level. If a user combines both sorting methods on a storefront, products are sorted based on the following sorting method priority:
+
+1. Manually specified sort order on a category level.
+2. Manually specified sort order on a product level. `0` by default. 
+
+If there is no sort order value on a category level, products are sorted by values on a product level.
```
Comment (traci707) [inline docs/api-docs/catalog/products-overview.md:1041]: If there is no sort order value on a category level, the Catalog API sorts products by values on a product level.

File: docs/api-docs/catalog/products-overview.md [hunk 4]
```diff
@@ -1030,6 +1026,32 @@ X-Auth-Token: {{ACCESS_TOKEN}}
...
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+<!-- theme:  -->
+
+### Note
+> Products with the same sort order value either on a category or a product level are sorted by `product id` as a second criterion.
```
Comment (traci707) [inline docs/api-docs/catalog/products-overview.md:1050]: The Catalog API sorts by `product id` as a second criterion for products with the same sort order value either on a category or a product level.

File: docs/api-docs/catalog/products-overview.md (hunk 1 above)
Comment (traci707) [reply docs/api-docs/catalog/products-overview.md:1036]: Feel free to rewrite my suggestion.  Would you say "product-level products"?

File: docs/api-docs/catalog/products-overview.md [hunk 5]
```diff
@@ -766,7 +735,6 @@ The following example shows how to add a modifier and a checkbox with a price ad
 Creating a checkbox with an adjuster requires two separate calls: one to create the checkbox and another one to add the adjuster. You can define adjusters within the `option_values` array, but `option_values` are not allowed in the request to create a checkbox modifier because creating a checkbox automatically generates two mandatory option values: `Yes` and `No`. Once you have created the checkbox along with its option values, you can update the modifier to add an adjuster.
```
Comment (traci707) [inline docs/api-docs/catalog/products-overview.md:736]: Once you have created the checkbox and its option values, you can update the modifier to add an adjuster.

**Review by traci707 — COMMENTED** [review]
Here are some suggestions.

**Comment by ektravel** [comment]
@uwikaiddi this PR is ready for final review

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**